app/containers/Products/product-schema.json

```
@@ -24,6 +24,59 @@
         "title": "CIIP (benchmarked) product"
       }
     },
+    "dependencies": {
+      "requiresEmissionAllocation": {
+        "oneOf": [
+          {
+            "properties": {
+              "requiresEmissionAllocation": {
+                "const": false
```

```suggestion
                "const": true
```
It's the opposite: if the product requires emission allocation, then there is no need to specify which electricity/heat emissions are added/substracted as the reported will do the calculation themselves.